### ISSUE TITLE
Generator - Extended point

### ISSUE THREAD

**Issue by vlopes11** [dusk-network/jubjub#31]
Most of the operations are performed with `ExtendedPoint` instead of `AffinePoint`, but only a generator in the affine form is available.

Create a constant `GENERATOR_EXTENDED` that will be a `GENERATOR` converted to `Extended`

This way the users of the API won't be obliged to do so.

```rust
pub const GENERATOR_EXTENDED: ExtendedPoint = ExtendedPoint {
    x: Fq::from_raw([
        0x4df7b7ffec7beaca,
        0x2e3ebb21fd6c54ed,
        0xf1fbf02d0fd6cce6,
        0x3fd2814c43ac65a6,
    ]),
    y: Fq::from_raw([
        0x0000000000000012,
        000000000000000000,
        000000000000000000,
        000000000000,
    ]),
    z: Fq::one(),
    t1: Fq::from_raw([
        0x4df7b7ffec7beaca,
        0x2e3ebb21fd6c54ed,
        0xf1fbf02d0fd6cce6,
        0x3fd2814c43ac65a6,
    ]),
    t2: Fq::from_raw([
        0x0000000000000012,
        000000000000000000,
        000000000000000000,
        000000000000,
    ]),
};
```

**Comment by CPerezz** [comment]
My suggestion would be to also implement `GENERATOR_NUMS`  also @vlopes11

**Comment by CPerezz** [comment]
Closed via #34
